jest.config.ts

```diff
@@ -0,0 +1,42 @@
+import type {Config} from '@jest/types';
+
+const config: Config.InitialOptions = {
+  verbose: true,
+  preset: 'ts-jest',
+  coverageDirectory: '__coverage__',
+  collectCoverage: true,
+  transformIgnorePatterns: [
+    'node_modules/(?!@patternfly|lodash-es|@console|@novnc|@spice-project|@popperjs|i18next)'
+  ],
+  coverageReporters: [
+    'json',
+    'lcov',
+    'text',
+    'text-summary'
+  ],
+  collectCoverageFrom: [
+    'packages/*/src/**/*.{js,jsx,ts,tsx}',
+    '!**/node_modules/**',
+    '!**/dist/**'
+  ],
+  testURL: 'http://localhost',
+  testRegex: '.*\\.(spec|test)\\.(ts|tsx|js|jsx)$',
```

Comment:
I'd suggest using a single way to name tests, i.e. just `spec` or `test` but not `spec|test`.

Reply:
Agreed, we should align... what does everyone think? 

I say `test` is always straight forward. `spec` always felt like "oh yeah, and that's the test"

Reply:
I think the `spec` comes from the history, honestly I'd rather keep both. Otherwise someone who is used to wrigin `.spec.ts` would fail to run the new tests.

Reply:
+1 for just `test`, AFAIK `spec` term originates from Ruby world (but I can be wrong).